src/vs/editor/browser/controller/editContext/native/nativeEditContext.ts

```diff
@@ -166,6 +171,7 @@ export class NativeEditContext extends AbstractEditContext {
 		this._register(editContextAddDisposableListener(this._editContext, 'characterboundsupdate', (e) => this._updateCharacterBounds(e)));
 		let highSurrogateCharacter: string | undefined;
 		this._register(editContextAddDisposableListener(this._editContext, 'textupdate', (e) => {
+			inputLatency.onInput();
```

Comment:
Wasn't totally sure about this one. Is there an equivalent to `input` for edit context?